src/contacts/Dockerfile

```diff
@@ -37,5 +37,8 @@ RUN pip install -r /app/requirements.txt
 # Add application code.
 ADD *.py /app/
 
+# Copy logging configuration for gunicorn
+COPY logging.conf /logging.conf
+
 # Start server using gunicorn
-CMD gunicorn -b :$PORT --threads 4 --log-level=info "contacts:create_app()"
+CMD gunicorn -b :$PORT --threads 4 --log-config /logging.conf --log-level=$LOG_LEVEL "contacts:create_app()"
```

Comment:
What happens if LOG_LEVEL isn't set? Will it cause a crash? Does it at least give a reasonable error message?

Maybe we should bake a default LOG_LEVEL value into the container to be overwritten by Kubernetes

Reply:
It seems like gunicorn sets the log level to info by [default](https://docs.gunicorn.org/en/stable/settings.html#loglevel) if the flag isn't specified. With `--log-level=$LOG_LEVEL`, if I don't specify the LOG_LEVEL anywhere, it seems like it also defaults to `info`.